lib/src/main/java/com/nextcloud/android/sso/model/ocs/OcsCapabilitiesResponse.java

```diff
@@ -0,0 +1,59 @@
+package com.nextcloud.android.sso.model.ocs;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * <p>This is a basic implementation for the <code>capabilities</code> endpoint which maps version and theming properties.<br>
+ * You can use it directly in combination with {@link OcsResponse} or extend it.</p>
+ * <p>Example usage with Retrofit:</p>
+ * <pre>
+ * {@code
+ * @GET("/ocs/v2.php/cloud/capabilities?format=json")
+ * Call<OcsResponse<OcsCapabilities>> getCapabilities();
+ * }
+ * </pre>
+ *
+ * @see <a href="https://docs.nextcloud.com/server/latest/developer_manual//client_apis/OCS/ocs-api-overview.html#capabilities-api">Capabilities API</a>
```

Comment:
there is a double slash in the url 🕵️ doesn't really matter though
```suggestion
 * @see <a href="https://docs.nextcloud.com/server/latest/developer_manual/client_apis/OCS/ocs-api-overview.html#capabilities-api">Capabilities API</a>
```